frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -420,10 +416,19 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
     }
 
     private void prepareStandardView(RenderRequest request) throws IOException {
-
         String searchtext = request.getParameter(KEY_SEARCH_TEXT);
-
         String searchfilter = request.getParameter(KEY_SEARCH_FILTER_TEXT);
+        List<Project> projectList = getFilteredProjectList(request);
+
+        request.setAttribute(PROJECT_LIST, projectList);
+        request.setAttribute(KEY_SEARCH_TEXT, searchtext);
```

Comment:
Same as above: does not change anything and is unused.

Same for `searchfilter`.